compiler/aarch64/codegen/OMRCodeGenerator.hpp

```diff
@@ -283,14 +283,25 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
     */
    void buildRegisterMapForInstruction(TR_GCStackMap *map);
 
+   /**
+    * @brief Answers if the method has a call or not
+    * @return true when the method has a call
+    */
+   bool hasCall() { return _flags.testAny(HasCall); }
+   /**
+    * @brief Sets the HasCall flag
+    */
+   void setHasCall() { _flags.set(HasCall); }
+
    TR_GlobalRegisterNumber _gprLinkageGlobalRegisterNumbers[TR::RealRegister::NumRegisters]; // could be smaller
    TR_GlobalRegisterNumber _fprLinkageGlobalRegisterNumbers[TR::RealRegister::NumRegisters]; // could be smaller
 
    private:
 
    enum // flags
       {
-      IsOutOfLineHotPath    = 0x00000200,
+      HasCall               = 0x00000200,
+      IsOutOfLineHotPath    = 0x00004000,
```

Comment:
Can these values be numbered a little more compactly?  Nothing functionally wrong with this, but they are interesting choices.  :-)

Reply:
I thought it might be better to use the same values as those used in `p` and `arm` versions.